test/src/org/labkey/test/tests/idri/FormulationsTest.java

```diff
@@ -177,9 +177,10 @@ protected void setupCompounds()
 
         log("Entering compound information");
         clickAndWait(Locator.linkWithText(COMPOUNDS_NAME));
-
-        SampleSetHelper helper = new SampleSetHelper(this);
-        helper.bulkImport(COMPOUNDS_HEADER + COMPOUNDS_DATA_1 + COMPOUNDS_DATA_2 + COMPOUNDS_DATA_3 + COMPOUNDS_DATA_4);
+        clickButton("Import More Samples");
+        ImportDataPage importDataPage = new ImportDataPage(getDriver());
+        importDataPage.setText(COMPOUNDS_HEADER + COMPOUNDS_DATA_1 + COMPOUNDS_DATA_2 + COMPOUNDS_DATA_3 + COMPOUNDS_DATA_4);
```

Comment:
I know this is preexisting, but it would be easier to read if it was: 
```suggestion
        importDataPage.setText( String.join('\n', COMPOUNDS_HEADER, COMPOUNDS_DATA_1, COMPOUNDS_DATA_2, COMPOUNDS_DATA_3, COMPOUNDS_DATA_4));
```
also would need to make the corresponding change to the constants too.